server/utils/azureAppInsights.ts

```diff
@@ -66,14 +66,14 @@ export function appInsightsMiddleware(): RequestHandler {
 export function addUserDataToRequests(envelope: EnvelopeTelemetry, contextObjects: ContextObject | undefined): boolean {
   const isRequest = envelope.data.baseType === Contracts.TelemetryTypeString.Request
   if (isRequest) {
-    const { username } = contextObjects?.['http.ServerRequest']?.res?.locals?.user || {}
-    const { activeCaseLoadId } = contextObjects?.['http.ServerRequest']?.res?.locals?.activeCaseLoadId || {}
```

Comment:
The core problem was here:
```
contextObjects?.['http.ServerRequest']?.res?.locals?.activeCaseLoadId || {}
```
because the `activeCaseLoadId` property is not directly on `res.locals` - it's on `res.locals.user`, so it should have been:
```
contextObjects?.['http.ServerRequest']?.res?.locals?.user?.activeCaseLoadId || {}
```

but I've done a little refactoring to reduce duplication re: getting the user object